### ISSUE TITLE
Device Available in Tuya Cloud, detectable via the wizard, but not via local scan detect nothing

### ISSUE THREAD

**Issue by NicolasDQualtech** [jasonacox/tinytuya#556]
Hi,, I did the wizard config. Everything went ok. It discovered my 2 devices.

But when I do local poll, i don't see anything...

Any idea?

I tried to disable the firewall to make sure it is not my problem.

Devices and my test PC and my home assistant are all on the same local network

Any Idea???

>> Saving list to devices.json
    2 registered devices saved

>> Saving raw TuyaPlatform response to tuya-raw.json

Poll local devices? (Y/n): y

Scanning local network for Tuya devices...
    0 local devices discovered

Polling local devices...
    [TY-16W                   ] Error: No IP found
    [water leak sensor        ] Error: No IP found

>> Saving device snapshot data to snapshot.json

Done.

**Comment by jasonacox** [comment]
What device is this?

* Turn off SmartLife or anything else that might be connecting to the devices. Some devices will stop responding once connected to a client.
* Battery powered Tuya devices will not show up (it keeps the WiFi offline to save power except when updating the Cloud).
* Tuya devices are "cloud" devices by default and the local API is used for setup, we just happen to exploit it for TinyTuya local access, but many of the devices do not have a robust WiFi system. Use the force scan function in TinyTuya to see if that helps:  

```
python3 -m tinytuya scan -f
```

**Comment by NicolasDQualtech** [comment]
Hi Jason,
I did try with Smartlife app closed.
Then ran the line you said. Python3 was not working, but python -m tinytuya scan -f was working. I found out some install was missing. I ran pip install netifaces, but it always failed saying C++ V14 was required. Version 14.42 is installed... I also ran pip install otherone, Then I was able to run the scan. Nothing was found, but Local tuya was able to detect my device, However, I always add an unknown error while trying to add the device...

I think I will give up on this... Thank you for your help, continue your great work
